Runtime/Common/ClientPathHelper.cs

```diff
@@ -59,7 +59,14 @@ private static string GenerateFullPath(this string path)
             {
                 path = Path.Combine(Application.persistentDataPath, path);
             }
-            return Path.GetFullPath(path);
+            try
+            {
+                return Path.GetFullPath(path);
+            }
+            catch (Exception)
+            {
+                return string.Empty;
```

Comment:
Would add a debug log message so the Exception is logged somewhere. Now, if this code has a problem, the exception details are completely swallowed and can't be retrieved in any way.

Reply:
It is in the higher level - it doesn't make sense to include debug logs in the helper, since later we will have a problem with helpers dependencies.